### PR TITLE
Studio release pr 2023 06 01

### PR DESCRIPTION
**Pull request by thaddmt**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-core/package.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-  "version": "2.1.20",
+  "version": "2.1.23",
   "main": "dist/index.js",
   "module": "dist/esm/index.js",
   "react-native": "dist/index.js",
   "exports": {
     ".": {
       "import": "./dist/esm/index.js",
       "require": "./dist/index.js"
+      "types": "./dist/types/index.d.ts"
```
Comment (thaddmt) [inline packages/react-core/package.json:11]: Please help me double verify this merge conflict was done correctly

File: packages/react-core/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "exports": {
     ".": {
       "import": "./dist/esm/index.js",
-      "require": "./dist/index.js"
+      "require": "./dist/index.js",
```
Comment (zchenwei) [inline packages/react-core/package.json:10]: Curious why the conflict could happen here? But it looks good to me

**Review by zchenwei — DISMISSED** [review]
🚢

**Review by ioanabrooks — APPROVED** [review]
LGTM

File: packages/react-core/package.json (hunk 2 above)
Comment (thaddmt) [reply packages/react-core/package.json:10]: Hm not sure, the conflict was between 

main
```
"import": "./dist/esm/index.mjs",
      "require": "./dist/index.js",
      "types": "./dist/types/index.d.ts"
```

studio/main
```
"import": "./dist/esm/index.js",
      "require": "./dist/index.js"
```

I tried fixing in the github merge tool but missed the comma

**Review by wlee221 — APPROVED** [review]
Remember to not squash ;)

**Review by 0618 — APPROVED** [review]

**Review by ioanabrooks — APPROVED** [review]

File: packages/react-core/package.json (hunk 2 above)
Comment (zchenwei) [reply packages/react-core/package.json:10]: Weird. That looks like an addition to me. 🤣

**Review by zchenwei — APPROVED** [review]

**Pull request merged**